tests_release.sh

```diff
@@ -37,18 +34,18 @@ run_tests() {
 
   # TensorFlow isn't a regular dependency because there are many different pip
   # packages a user might have installed.
-  pip install tensorflow
+  pip install tensorflow==2.0.0-beta1
```

Comment:
Not sure if we should pin to an specific beta version.

Reply:
I agree. How should I proceed? 

Not specifying a version at all leads to installing r1.13/r1.14. There are different tests which fail when using one of these two versions of tensorflow (compare with r2.0). Should I have some way of version checking in these tests that differ?

Not specifying a beta version seems incompatible with pip, which wants either b1 or b0 for r2.0.

Reply:
Changed in 90a8f81 to just install the latest version of tf.